problem-spring-web-autoconfigure/pom.xml

```diff
@@ -63,12 +63,17 @@
             <version>${spring-security.version}</version>
             <optional>true</optional>
         </dependency>
-
         <dependency>
             <groupId>org.zalando</groupId>
             <artifactId>problem-spring-web</artifactId>
             <version>${project.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <version>${spring-boot.version}</version>
+            <scope>test</scope>
```

Comment:
You need to exclude 

- `org.hamcrest:java-hamcrest`
- `ch.qos.logback:logback-classic`